com.sap.cloud.lm.sl.cf.web/src/main/webapp/WEB-INF/services-context.xml

```diff
@@ -8,10 +8,15 @@
     <bean id="fileSystemFileStorage" class="com.sap.cloud.lm.sl.cf.web.configuration.bean.factory.FileSystemFileStorageFactoryBean">
         <property name="serviceName" value="deploy-service-fss" />
     </bean>
+    
+    <bean id="objectStoreFileStorage" class="com.sap.cloud.lm.sl.cf.web.configuration.bean.factory.ObjectStoreFileStorageFactoryBean">
```

Comment:
You could remove the bean and use directly the java spring bean configuration: check this https://cloud.spring.io/spring-cloud-connectors/spring-cloud-connectors.html

Reply:
Refactored the file storage creation into a java spring bean config